main.go

```diff
@@ -370,9 +370,6 @@ func main() {
 			signedAPKPaths = append(signedAPKPaths, fullPath)
 		}
 
-		if err != nil {
```

Comment:
This was checking an `err` instance assigned waaay earlier, I don't think we need this here.